### ISSUE TITLE
Error in js

### ISSUE THREAD

**Issue by morzius** [Studio-42/elFinder#1250]
I got this error in file elfinder.full.js:179
Uncaught ReferenceError: Invalid left-hand side expression in postfix operation

same if i try to use elfinder.min.js

**Comment by nao-pon** [comment]
@morzius What version are you using? Please try newer version.

**Comment by chebotaryov** [comment]
@morzius and I tried to integrate stable versions 2.1.7 and 2.1.8 with ckEditor.

**Comment by nao-pon** [comment]
2.1.8 [elfinder.full.js L.179](https://github.com/Studio-42/elFinder/blob/fd32822ed4991c03865f58dfbffa773c95ebe539/js/elfinder.full.js#L179) is
```javascript
		 * @type Array
```
L. 174-
```javascript
		/**
		 * Copied/cuted files hashes
		 * Prevent from remove its from cache.
		 * Required for dispaly correct files names in error messages
		 *
		 * @type Array
		 **/
		remember = [],
```
It does not look like there is a problem there.

**Comment by morzius** [comment]
i downloaded last version from github, 2.1.8, and in editor my 179 line is same. But if i open this js through browser i got this:

elfinder.full.js:179 Uncaught ReferenceError: Invalid left-hand side expression in postfix operation

and line 179 looks like
`if(blobSlice&&blobSize>BYTES_PER_CHUNK){start=0;end=BYTES_PER_CHUNK;chunks=-1;total=Math.floor(blobSize/BYTES_PER_CHUNK);totalSize+=blobSize;chunked[chunkID]=0;while(start<=blobSize){chunk=blob[blobSlice](start,end);chunk._chunk=blob.name+'.'+++chunks+'_'+total+'.part';chunk._cid=chunkID;chunk._range=start+','+chunk.size+','+blobSize;chunked[chunkID]++;if(size){c++;}`

**Comment by nao-pon** [comment]
@morzius Please check with elfinder.full.js. And tell me that line number.

**Comment by nao-pon** [comment]
@morzius What about this [elfinder.full.js](http://hypweb.net/elFinder-nightly/demo/Demo/Downloads/debug/elfinder.full.js).

**Comment by morzius** [comment]
Looks like its our web server, its have js compressor or something

elfinder.full.js appears in different ways in editor and in browser

**Comment by nao-pon** [comment]
OK, Maybe fixed it. Can you use this [elfinder.full.js](https://gist.github.com/nao-pon/fe9681ab43bf463bcbcd/archive/a1f7edec6b962c57f88b73c109a35c8d6dced91f.zip)?

- Source: https://gist.github.com/nao-pon/fe9681ab43bf463bcbcd
- Download: https://gist.github.com/nao-pon/fe9681ab43bf463bcbcd/archive/a1f7edec6b962c57f88b73c109a35c8d6dced91f.zip

**Comment by morzius** [comment]
i just put elfinder folder to ignore for this compressor, thank you anyway

**Comment by nao-pon** [comment]
I think that has been fixed probably by @3caf56beb14716447083f6b40924c161a401740a .
